google/cloud/bigtable/internal/data_connection_impl.cc

```diff
@@ -31,6 +33,33 @@ DataConnectionImpl::DataConnectionImpl(
           std::move(options),
           bigtable::internal::DefaultDataOptions(DataConnection::options()))) {}
 
+Status DataConnectionImpl::Apply(std::string const& app_profile_id,
+                                 std::string const& table_name,
+                                 bigtable::SingleRowMutation mut) {
+  google::bigtable::v2::MutateRowRequest request;
+  request.set_app_profile_id(app_profile_id);
+  request.set_table_name(table_name);
+  mut.MoveTo(request);
+
+  auto idempotent_policy = idempotency_policy();
+  bool const is_idempotent = std::all_of(
+      request.mutations().begin(), request.mutations().end(),
+      [&idempotent_policy](google::bigtable::v2::Mutation const& m) {
+        return idempotent_policy->is_idempotent(m);
+      });
+
+  auto sor = google::cloud::internal::RetryLoop(
```

Comment:
nit: I think we avoid names like this.  `response` may be better.  Your call.

Reply:
hmm.. I think I will keep using `sor` but feel self-conscious about it.